src/repr/src/explain/tracing.rs

```diff
@@ -315,6 +319,62 @@ impl field::Visit for ExtractStr {
     fn record_debug(&mut self, _field: &tracing::field::Field, _value: &dyn std::fmt::Debug) {}
 }
 
+/// A [`subscriber::Subscriber`] implementation that delegates to the default
+/// [`Dispatch`] instance.
+#[allow(missing_debug_implementations)]
+
+pub struct DelegateSubscriber {
+    inner: Dispatch,
+}
+
+impl Default for DelegateSubscriber {
+    fn default() -> Self {
+        let default = tracing::dispatcher::get_default(Clone::clone);
+
+        let inner = if default.downcast_ref::<Registry>().is_none() {
+            Registry::default().into() // we need at least a simple registry
```

Comment:
when is this case actually hit?

Reply:
I was just future-proofing against scenarios where this runs without an `mz_ore::tracing::configure()` call (this happens in `*.td` tests for example).